system-test/storage.ts

```diff
@@ -3140,26 +3138,6 @@ describe('storage', () => {
     );
   }
 
-  async function isFilePublicAsync(file: File) {
```

Comment:
It might be good to keep this because the `makePublic()` function is accomplishing that a certain way, and this makes sure we did it that way.

Reply:
Agree, thank you
Moved `isPublic()` test to a scenario without credentials.